File: python_work/220711/Friend.py
class Friend:
    
    def __init__(self, name, phone):
        self.name=name
        self.phone=phone
    
    def get_name(self):
        return self.name 
    
    def get_phone(self):
        return self.phone 
    
    def set_phone(self,phone):
        self.phone=phone 
    
    def show_info(self):
        print("이름: {}\n전화번호: {}".format(self.name, self.phone))

File: java_work/ex0715/src/ex0715/Friend.java
package ex0715;

public class Friend {

	private String name;
	private String phone;
	
	public Friend(String name, String phone) {
		super();//사실 상위 클래스에 object 클래스가 있어서(생략됨) super가 자동생성됨 
		this.name = name;
		this.phone = phone;
	}
	
	public void showInfo() {
		System.out.println("이름 = "+name);
		System.out.println("전화 = "+phone);
	}

	@Override
	public String toString() {
		return "Friend [name=" + name + ", phone=" + phone + "]";
	}
	
}
